pyaerocom/colocation_auto.py

```diff
@@ -223,7 +225,8 @@ def __init__(self, model_id=None, obs_id=None, obs_vars=None,
         self.model_vert_type_alt = model_vert_type_alt
         self.read_opts_ungridded = read_opts_ungridded
         self.obs_ts_type_read = obs_ts_type_read
-
+        self.gridded_reader_id = gridded_reader_id
+        self.data_id = data_id
```

Comment:
Is data_id used anywhere?

Reply:
Good catch, used it for testing. WIll remove.